node/tests/initialization_test.rs

```diff
@@ -260,7 +260,7 @@ fn requested_chain_meets_different_db_chain_and_panics_integration() {
         r"ERROR: PanicHandler: src(/|\\)actor_system_factory\.rs.*- Database with a wrong chain name detected; expected: {}, was: eth-mainnet",
         &chain_literal
     );
-    node.wait_for_log(&regex_pattern, Some(1000));
+    node.wait_for_log(&regex_pattern, Some(5000));
```

Comment:
Aspon kdybychom mohli tohle probrat s Danem. Myslim, ze by stalo za to mit environment variable, neverejny parametr, ktery kdyz zadame v testu, tak Node preskoci inicializaci te ip_country databaze apod.